addon/mocks/exposed-request-functions.js

```diff
@@ -58,7 +58,7 @@ export function mockTeardown() {
 export function mockFindRecord(...args) {
   let modelName;
 
-  Ember.assert(`[ember-data-factory-guy] mockFind requires at least a model
+  Ember.assert(`[ember-data-factory-guy] mockFindRecord requires at least a model
```

Comment:
This could be confusing if someone is using `mockFind`, but at least there will also be a deprecation notice.